interp/interp_eval_test.go

```diff
@@ -495,6 +495,7 @@ func TestEvalSliceExpression(t *testing.T) {
 		{src: `a := []int{0,1,2,3}[1::4]`, err: "1:49: 2nd index required in 3-index slice"},
 		{src: `a := []int{0,1,2,3}[1:3:]`, err: "1:51: 3rd index required in 3-index slice"},
 		{src: `a := []int{0,1,2}[3:1]`, err: "invalid index values, must be low <= high <= max"},
+		{src: `a := []int{0,1,2}; append(a)`, res: "[0 1 2]"},
```

Comment:
perhaps this can move to `TestEvalBuiltin` with the other `append` tests. We should also add a test for `append(nil)`, WDYT?

Reply:
yes, done